src/components/renderers/layouts/categories/categorization-directive.ts

```diff
@@ -7,20 +7,22 @@ class CategorizationDirective implements ng.IDirective {
     restrict = 'E';
     template = `
     <jsonforms-layout>
-        <div class="row">
-            <div class="col-sm-100">
-                <uib-tabset>
-                    <uib-tab
-                        heading="{{category.label}}"
-                        ng-repeat="category in vm.uiSchema.elements"
-                        select="vm.changeSelectedCategory(category)">
-                        <fieldset ng-if="vm.selectedCategory===category">
-                            <jsonforms-inner ng-repeat="child in category.elements"
-                                             ui-schema="child" >
-                            </jsonforms-inner>
-                        </fieldset>
-                    </uib-tab>
-                </uib-tabset>
+        <div class="jsf-categorization">
```

Comment:
Placing the class attribute onto the existing elements didn't work with the vertical/horizontal layout, that's why also introduced the additional element here as well, but I didn't actually give it a try. Will do.